tssc/step_implementers/push_container_image/skopeo.py

```diff
@@ -1,5 +1,37 @@
-"""
-Step Implementer for the create-container-image step for Buildah.
+"""Step Implementer for the create-container-image step for Skopeo.
+
+Step Configuration
+------------------
+
+Step configuration expected as input to this step.
+Could come from either configuration file or
+from runtime configuration.
+
+| Configuration Key | Required? | Default  | Description
+|-------------------|-----------|----------|-----------
+| `destination`     | True      |          | Destination to push image to
+| `src-tls-verify`  | True      | `'true'` | Whether to very TLS for source of image
+| `dest-tls-verify` | True      | `'true'` | Whether to verify TLS for destination of image
+
+Expected Previous Step Results
+------------------------------
+
+Results expected from previous steps that this step requires.
+
+| Step Name                | Result Key       | Description
+|--------------------------|------------------|------------
+| `generate-metadata`      | `image-tag`      | Tag to push image with
+| `create-container-image` | `image-tar-file` | Local tar file of image to push
+
+Results
+-------
+
+Results output by this step.
+
+| Result Key  | Description
+|-------------|------------
+| `image-tag` | Pushed destination image tag
+
```

Comment:
@itewk Missing example like other steps

Reply:
i got tired of placing examples and didn't seem the as useful. if people want to add in some later we can. for now i think we will just do this as is.